src/data-generator/declare.ts

```diff
@@ -34,9 +34,16 @@ import {
   FETCH_REGISTRATION_QUERY,
   SEARCH_EVENTS
 } from './queries'
+import { join } from 'path'
+import { readFileSync } from 'fs'
 
 const HOME_BIRTH_WEIGHT = 0.2
 const HOME_DEATH_WEIGHT = 0.2
+const NUMBER_OF_ATTACHMENTS_PER_RECORD = 4
```

Comment:
Do we always want the attachments to be generated?

Reply:
I'm not sure to be honest. I guess having at least a few per record would reflect the reality. I'll make this configurable with 2 as default if that sounds ok to you

Reply:
Okay sounds reasonable. it's just that I was concerned about increasing the running time of the data-generator by including attachments.